database/data/seed_data.js

```diff
@@ -0,0 +1,120 @@
+const lorem = require('lorem-ipsum');
+
+const options = require('../lorem_options');
+const dogBreeds = require('../dog_breeds.js');
+
+
+// ex. randomInt(1,5) => generates 1-5 inclusive
+let randomInt = (min, max) => {
+  return Math.floor(Math.random() * (max - min + 1)) + min;
+}
+
+let breedTypes = ['Herding', 'Sporting', 'Non-Sporting', 'Working', 'Hounds', 'Terriers', 'Toy'];
+
+let breedInfoArray = []
+
+//will generate random numbers, categories or ipsum data depending on breed attribute
+for (let i = 1; i <=100; i++) {
+
+  let randomWeightMin = randomInt(5, 75);
+  let randomWeightMax = randomWeightMin + randomInt(5, 50);
+
+  let randomHeightMin = randomInt(6, 48);
+  let randomHeightMax = randomHeightMin + randomInt(1, 12);
+
+  //generate array of 0-2 ipsum otherNames of 1-2 words
+  let numberOfOtherNames = randomInt(0,3);
+  let otherNameList = [];
+  for (let i = 0; i < numberOfOtherNames; i++) {
+    otherNameList.push(lorem.loremIpsum({count:randomInt(1,2), units: 'words'}))
+  }
+
+  let otherNameString = otherNameList.join(', ')
+
+  let numberOfConcerns = randomInt(1,8);
+  let majorConcernsList = [];
+  let minorConcernsList = [];
+  for (let i = 0; i < numberOfConcerns; i++) {
+    majorConcernsList.push(lorem.loremIpsum({count:randomInt(1,3), units: 'words'}));
+    minorConcernsList.push(lorem.loremIpsum({count:randomInt(1,2), units: 'words'}))
+  }
+  let majorConcernsString = majorConcernsList.join(', ')
+  let minorConcernsString = minorConcernsList.join(', ')
+
+  let numberOfOccasionallySeen = randomInt(1,3);
+  let occasionallySeenList = []
+  for (let i = 0; i < numberOfOccasionallySeen; i++) {
+    occasionallySeenList.push(lorem.loremIpsum({count:randomInt(1,2), units: 'words'}))
+  }
+  let occasionallySeenString = occasionallySeenList.join(', ')
+
+
+  let numberOfSuggestedTests = randomInt(1,4);
+  let suggestedTestsList = [];
+  for (let i = 0; i < numberOfSuggestedTests; i++) {
+    suggestedTestsList.push(lorem.loremIpsum({count:randomInt(1,2), units: 'words'}))
+  }
+  let suggestedTestsString = suggestedTestsList.join(', ')
+
```

Comment:
if you want to DRY your code here, could create a helper function that generates lists of words and gives you an array back. From looking here it seems like the only different parameters you would have to pass in would be number of items in the array, and the number of words in each item (looking up from here at other names, major/minor concerns, occasionally seen, suggested tests)